src/android/toga_android/app.py

```diff
@@ -41,21 +46,16 @@ def __init__(self, interface):
         self.interface._impl = self
 
     def create(self):
-        # Connect this app to the PythonActivity
         self._listener = TogaApp(self)
 
-        # Set the Python activity listener to be this app.
-        self.native = PythonActivity.setListener(self._listener)
-
-        self.startup()
```

Comment:
I'm a little surprised you can live without this one, but we can always add it back in if we need to .